Library/Formula/syntaxerl.rb

```diff
@@ -0,0 +1,20 @@
+class Syntaxerl < Formula
+  homepage "https://github.com/ten0s/syntaxerl"
```

Comment:
Sorry, we've added a new description (`desc`) attribute. Can you add one? Thanks!